From be3238a931bca3419279f9bdca2e249e97831e43 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 21 Feb 2024 13:27:48 +0100
Subject: [PATCH 0625/1695] plat-stm32mp2: Disable EXTI for STM32MP215x boards

The EXTI driver doesn't support the STM32MP21 SoC family yet.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d7c807c8b6a7f2fdee7dacbde0842c51cedc3a77 plat-stm32mp2: Disable EXTI for STM32MP215x boards]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ib77dc80e9e50c94e4b128afaa20a36d627286809
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/361804
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index b5fb5df81..8112b34c5 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -116,6 +116,10 @@ endif
 CFG_TEE_CORE_NB_CORE ?= 2
 CFG_STM32MP_OPP_COUNT ?= 3
 
+ifeq ($(CFG_STM32MP21),y)
+CFG_STM32_EXTI ?= n
+endif
+
 CFG_STM32_BSEC3 ?= y
 CFG_STM32_BSEC_WRITE ?= y
 CFG_STM32_EXTI ?= y
-- 
2.47.2

